ds/list/list_test.go

```diff
@@ -377,6 +377,22 @@ func TestList_LRem7(t *testing.T) {
 	num, err = list.LRem(key, math2.MinInt, []byte("b"))
 	assertions.EqualError(err, ErrMinInt.Error())
 	assertions.Equal(0, num)
+
+	num, err = list.LRem(key, 0, []byte("item_not_exists"))
```

Comment:
[golangci] reported by reviewdog 🐶
ineffectual assignment to err (ineffassign)

Raw Output:
ds/list/list_test.go:381:7: ineffectual assignment to err (ineffassign)
	num, err = list.LRem(key, 0, []byte("item_not_exists"))

Reply:
thanks for your comments, have just updated.